product_yahoo_auction_sst/models/delivery_carrier_size.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class DeliveryCarrierSize(models.Model):
+    _name = 'delivery.carrier.size'
+
+    size = fields.Char(
```

Comment:
Why not just name the field `name` instead of manipulating with name_get method?